modules/web/client/src/main/scala/observe/ui/Icons.scala

```diff
@@ -156,11 +164,14 @@ object Icons:
     faFileCheck,
     faFileCross,
     faGears,
+    faMinus,
     faMoon,
     faPause,
     faPlay,
+    faPlus,
     faRectangleList,
     faReply,
+    faSignOutAlt,
```

Comment:
I forgot to add this before, but the icon was still working. So I'm not sure what this list actually does

Reply:
Strange that should be needed to actually load the esm module to get the icon